WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+
+class CommentContentTableViewCell: UITableViewCell, NibReusable {
+
+    // determines the state of the like button.
+    enum LikeState {
```

Comment:
Maybe I'm not seeing the big picture yet, but couldn't you just use `isLiked` from the `Comment`?

Reply:
Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/17109/commits/05d5a31e08a13244e2a493f9cf2aea0b40f07f88.